commit 245c8cb8fe488fd25dea7e61885def4612759829
Author: Jordan Paige Hendricks <jordan.hendricks@joyent.com>
Date:   2019-09-04T14:00:01+00:00 (5 weeks ago)
    
    OS-7962 enable hotplug service by default
    Reviewed by: Rob Johnston <rob.johnston@joyent.com>
    Reviewed by: John Levon <john.levon@joyent.com>
    Approved by: John Levon <john.levon@joyent.com>

diff --git a/usr/src/cmd/svc/profile/generic.xml b/usr/src/cmd/svc/profile/generic.xml
index 1920d1e8a6..61232545dd 100644
--- a/usr/src/cmd/svc/profile/generic.xml
+++ b/usr/src/cmd/svc/profile/generic.xml
@@ -23,6 +23,8 @@
     Copyright 2010 Sun Microsystems, Inc.  All rights reserved.
     Use is subject to license terms.
 
+    Copyright 2019 Joyent, Inc.
+
     The purpose of the limited_net profile is to provide a set of
     active services that allow one to connect to the machine via ssh
     (requires sshd). The services which are deactivated here are those
@@ -93,6 +95,9 @@
   <service name='system/rcap' version='1' type='service'>
     <instance name='default' enabled='true'/>
   </service>
+  <service name='system/hotplug' version='1' type='service'>
+    <instance name='default' enabled='true'/>
+  </service>
   <service name='network/rpc/bind' version='1' type='service'>
     <instance name='default' enabled='true'/>
   </service>
